drivers/flash/flash_stm32_qspi.c

```diff
@@ -482,6 +567,17 @@ static int flash_stm32_qspi_write(const struct device *dev, off_t addr,
 
 	qspi_lock_thread(dev);
 
+#if CONFIG_STM32_MEMMAP
+	if (stm32_qspi_is_memory_mapped(dev)) {
+		/* Abort ongoing transfer to force CS high/BUSY deasserted */
+		ret = stm32_qspi_abort(dev);
+		if (ret != 0) {
+			LOG_ERR("Failed to abort memory-mapped access before write");
+			goto end;
+		}
```

Comment:
suggestion : `LOG_DBG("Abort Memory-mapped before write");`

Reply:
That feels much less clear to me. For one it doesn't even mention that something went wrong, and it's also not the memory-mapping that's being aborted, just a latent read access, if any.

Reply:
I think the suggestion was to put it after, once abort is completed.

Reply:
> I think the suggestion was to put it after, once abort is completed.

@arbrauns Can you check this comment ?

Reply:
Ah, that makes sense, but then that `LOG_DBG()` would trigger on every single flash write/erase - would that make sense?

Reply:
There's already a LOG_DBG on flash_stm32_qspi_read so, that would be coherent.